src/core/src/pass/serialize.cpp

```diff
@@ -35,6 +35,27 @@ std::string join(const Container& c, const char* glue = ", ") {
     return oss.str();
 }
 
+std::string dimension_to_str(const Dimension& dim) {
+    if (dim.is_dynamic() && !dim.get_interval().has_upper_bound() && (dim.get_min_length() == 0))
```

Comment:
Looks like `dim.is_dynamic()` is not needed as we still check bounds.

Reply:
`(dim.get_min_length() == 0)` -> `dim.get_min_length() == 0`

Reply:
@jane-intel what do you think about `dim.get_interval().has_lower_bound()` to avoid this manual checks for zero?

Reply:
I removed this function and used << operator from PartialShape instead. And it uses has_lower_bound() instead of check for zero. @GlebKazantaev please, review.